yamale/tests/test_functional.py

```diff
@@ -176,6 +182,10 @@ def test_bad_nested():
     assert count_exception_lines(nested['schema'], nested['bad']) == 4
 
 
+def test_bad_nested_issue_54():
```

Comment:
I'm not a fan of this format of checking the number of lines in the exception message and would like to change the existing tests. For example, you're missing the `nested_issue_54.yaml` file, but the test passes. This means that the "file not found" exception message has the same number of lines as what you're raising.

Reply:
I agree that test_functional.py looks complex and just counting errors is not enougth, that why I also did test in test_command_line.py where we can easily check errors

Reply:
I'd rather not add more things that I'll have to change later. Can you change this to match the format of the tests below that use `match_exception_lines()`? You can create a separate schema file instead of reusing `nested.yaml` if you want to limit the number of errors in your output. Thanks!